lib/maplibre-compose/src/commonMain/kotlin/dev/sargunv/maplibrecompose/core/expression/ExpressionsDsl.kt

```diff
@@ -29,6 +34,27 @@ public object ExpressionsDsl {
   /** Creates a literal expression for a [Dp] value. */
   public fun const(dp: Dp): Expression<DpValue> = Expression.ofFloat(dp.value).cast()
 
+  /**
+   * Creates a literal expression for a specified [TextUnit] value. If [textUnit] is in `em`, it is
+   * considered relative to `16.sp`.
+   */
+  public fun const(textUnit: TextUnit, fontScale: Float): Expression<TextUnitValue> =
```

Comment:
Why should it be possible for the user to pass the `Density`, yet alone the `fontScale` himself at all?

Reply:
In the non composable function, because LocalDensity isn't available

in the composable version, prob not necessary yeah.

Reply:
(Uf, wrong button and the typed text is gone)

I overread that one function is composable, the other not. But I'd say, this would only ever be called in composable functions, wouldn't it?